### PR TITLE
sync ua & en translations + corrected spelling

### PR DESCRIPTION
**Pull request by asya982**
- Synchronized all ua translation with en ones
- Added missing translations to ua object
- Fixed some spellings in keys

### CODE REVIEW

**Review by GeorgeShvab — APPROVED** [review]

**Review by D1nviz — APPROVED** [review]

File: src/constants/translations/ua/filters.json [hunk 1]
```diff
@@ -9,5 +9,5 @@
     	}
     },
 	"filtersListTitle": "Фільтри",
-	"filtersLevelsLable": "Рівні"
+	"filtersLevelsLabel": "Рівні"
```
Comment (mhevyk) [inline src/constants/translations/ua/filters.json:12]: I suggest refactoring to `filterLevelsLabel`

File: src/constants/translations/ua/common.json [hunk 2]
```diff
@@ -42,23 +45,52 @@
...
+    "subject": "Будь ласка, виберіть предмет",
+    "proficiencyLevel": "Будь ласка, виберіть принаймні один рівень знань",
     "youtubeLink": "Будь ласка, надайте посилання на YouTube відео"
   },
   "levels": {
     "beginner": "Початковий",
     "intermediate": "Середній",
     "advanced": "Високий",
-    "testPreparation": "Підготовка до тесту",
+    "test preparation": "Підготовка до тесту",
```
Comment (mhevyk) [inline src/constants/translations/ua/common.json:93]: Why do we use space here? I think we should stick to `camel case` convention

**Review by mhevyk — CHANGES_REQUESTED** [review]

**Review by t-valentyna — APPROVED** [review]

File: src/containers/guest-home-page/reset-password/ResetPassword.jsx [hunk 3]
```diff
@@ -32,7 +32,7 @@ const ResetPassword = ({ resetToken, openModal }) => {
         <ImgTitleDescription
           img={imgSuccess}
           style={styles}
-          title={t('login.sucsessReset')}
```
[the remark this answers is not in the document]
Comment (asya982) [reply src/containers/guest-home-page/reset-password/ResetPassword.jsx:35]: It's already fixed)

File: src/constants/translations/ua/login.json [hunk 4]
```diff
@@ -1,11 +1,18 @@
...
-  "joinUs": "Приєднуйтесь до нас безкоштовно"
+  "joinUs": "Приєднуйтесь до нас безкоштовно",
+  "enterEmail": "Введіть адресу електронної пошти, пов’язану з вашим обліковим записом, і ми надішлемо вам посилання для зміни пароля.",
+  "sendPassword": "Надішліть мені посилання для зміни пароля",
+  "backToLogin": "Назад до Логіну",
+  "passwordReset": "Скидання пароля",
+  "weSentEmail": "Ми надіслали вам електронний лист із посиланням для зміни пароля на адресу: \n",
+  "emailArrive": ". Якщо електронний лист не надійшов, дайте йому кілька хвилин і перевірте ще раз. Якщо його ще немає, перевірте папку спаму.",
+  "newPassword": "Створіть новий пароль",
+  "resetPasswordDesc": "Ваш новий пароль має відрізнятися від попередніх паролів.",
```
[the remark this answers is not in the document]
Comment (asya982) [reply src/constants/translations/ua/login.json:15]: What name exactly?

[the remark this answers is not in the document]
Comment (mhevyk) [reply src/constants/translations/ua/login.json:15]: `resetPasswordDesc`

File: src/constants/translations/ua/titles.json [hunk 5]
```diff
@@ -1,3 +1,4 @@
 {
-  "confirmTitle": "Підтведрдіть вихід"
+  "confirmTitle": "Підтведрдіть вихід",
```
[the remark this answers is not in the document]
Comment (asya982) [reply src/constants/translations/ua/titles.json]: Fixed!

File: src/constants/translations/ua/footer.json [hunk 6]
```diff
@@ -1,6 +1,6 @@
 {
     "allRightsReserved": "© Всі права захищені  Space to Study",
     "privacyPolicy": "Політика конфіденційності",
-    "termOfUse": "Термін використання"
+    "termOfUse": "Умови використання"
```
[the remark this answers is not in the document]
Comment (asya982) [reply src/constants/translations/ua/footer.json]: fixed! Thank you

File: src/constants/translations/ua/filters.json (hunk 1 above)
Comment (asya982) [reply src/constants/translations/ua/filters.json]: fixed!

File: src/constants/translations/ua/common.json (hunk 2 above)
Comment (asya982) [reply src/constants/translations/ua/common.json]: fixed, but a bug was discovered) will create an issue for it

File: src/constants/translations/ua/edit-profile.json [hunk 7]
```diff
@@ -7,6 +7,14 @@
...
       "uploadTitle": "Завантажити нове зображення",
       "uploadDesc": "Максимальний розмір файлу - 5 МБ",
+      "informationTitle": "Персональна інформація",
+      "informationDesc": "Ваше ім’я може з’являтися у Space2Study під час згадування.",
+      "locationTitle": "Локація",
+      "locationDesc": "Повідомте учасникам, де ви знаходитесь.",
+      "professionalHeadline": "Професійний заголовок",
+      "professionalStatus": "Коротко опишіть своє професійне резюме. Максимум 200 символів.",
+      "languagesTitle": "Мова",
+      "languagesDesc": "Виберіть мову, якоб ви б хотіли навчатися та співпрацювати.",
```
[the remark this answers is not in the document]
Comment (asya982) [reply src/constants/translations/ua/edit-profile.json]: thank you, fixed)

File: src/constants/translations/ua/error-page.json [hunk 8]
```diff
@@ -1,4 +1,18 @@
...
+  "404": {
+    "title": "Сторінку не знайдено",
+    "description": "Вибачте, але сторінка, яку ви намагалися знайти,\nне існує. Пропонуємо вам перейти на головну сторінку."
+  },
+  "500": {
+    "title": "Внутрішня помилка сервера",
+    "description": "На жаль, на сервері сталася неочікувана помилка, через яку він не міг виконати запит. Пропонуємо вам перейти на головну сторінку. Повідомити про помилку."
+  },
+  "400": {
+    "title": "Поганий запит",
```
[the remark this answers is not in the document]
Comment (asya982) [reply src/constants/translations/ua/error-page.json]: Thanks for good observasion, fixed!

**Review by iva-stasia — APPROVED** [review]

File: src/constants/translations/ua/login.json (hunk 4 above)
[the remark this answers is not in the document]
Comment (asya982) [reply src/constants/translations/ua/login.json]: thank you for you observation, fixed it

File: src/constants/translations/ua/common.json (hunk 2 above)
Comment (mhevyk) [reply src/constants/translations/ua/common.json]: Sure

**Review by mhevyk — APPROVED** [review]

**Review by GeorgeShvab — APPROVED** [review]

**Review by byavd — APPROVED** [review]

**Pull request merged**